cvat/apps/engine/cron.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import importlib
+import logging
+import os
+import os.path as osp
+from contextlib import suppress
+from datetime import timedelta
+from pathlib import Path
+
+from django.db.models import QuerySet
+from django.utils import timezone
+
+from cvat.apps.dataset_manager.util import (
+    ExportCacheManager,
+    LockNotAvailableError,
+    get_export_cache_lock,
+)
+from cvat.apps.dataset_manager.views import (
+    EXPORT_CACHE_LOCK_ACQUISITION_TIMEOUT,
+    EXPORT_CACHE_LOCK_TTL,
+    get_export_cache_ttl,
+    log_exception,
+)
+from cvat.apps.engine.log import ServerLogManager
+from cvat.apps.engine.models import Job, Project, Task
+
+
+class FileIsBeingUsedError(Exception):
+    pass
+
+
+def clear_export_cache(file_path: str, logger: logging.Logger) -> None:
+    try:
+        with get_export_cache_lock(
+            file_path,
+            block=True,
+            acquire_timeout=EXPORT_CACHE_LOCK_ACQUISITION_TIMEOUT,
+            ttl=EXPORT_CACHE_LOCK_TTL,
+        ):
+            if not osp.exists(file_path):
+                raise FileNotFoundError(f"Export cache file {file_path} doesn't exist")
+
+            parsed_filename = ExportCacheManager.parse_file_path(file_path)
+            cache_ttl = get_export_cache_ttl(parsed_filename.instance_type)
+
+            if timezone.now().timestamp() <= osp.getmtime(file_path) + cache_ttl.total_seconds():
+                logger.info("Cache file '{}' is recently accessed".format(file_path))
+                raise FileIsBeingUsedError
+
+            os.remove(file_path)
+            logger.debug(f"Export cache file {file_path!r} successfully removed")
+    except LockNotAvailableError:
+        logger.info(f"Failed to acquire export cache lock for the file: {file_path}.")
+        raise
```

Comment:
Does it make sense to raise here? You've already logged the message, so what is propagating the exception going to achieve?